dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/servlets/SipApplicationSessionImpl.java

```diff
@@ -1800,4 +1806,36 @@ public void setSessionKeyBaseTargeting(String skbt){
 		}
 		m_sessionKeyBaseKey = skbt;
 	}
+	
+	public Iterator getSessions(String protocol, boolean b) {
```

Comment:
Ah, you use `protocol` here. Could you update  `b` to `create`?

Reply:
changed b to protocol